larray/example.py

```diff
@@ -77,3 +83,20 @@ def load_example_data(name):
     if name not in AVAILABLE_EXAMPLE_DATA.keys():
         raise ValueError("example_data must be chosen from list {}".format(list(AVAILABLE_EXAMPLE_DATA.keys())))
     return la.Session(AVAILABLE_EXAMPLE_DATA[name])
+
+
+def get_example_templates_dir():
```

Comment:
get_example_excel_templates_dir? (or something else with excel in the name)

I know this is long but there is a significant probability we add other kinds of templates one day.

btw: do we really need a function for this? Why can't we just export EXAMPLE_EXCEL_TEMPLATES_DIR?